content/docs/start/experiments.md

```diff
@@ -2,293 +2,225 @@
-regular pipeline by committing it in our Git branch:
-
-```dvc
-$ git add dvc.lock params.yaml prc.json roc.json scores.json
-$ git commit -a -m "Preserve best random forest experiment"
-```
+## Running the experiment by setting parameters
 
-## Sharing experiments
+Now let's do some more experimentation.
 
-After committing the best experiments to our Git branch, we can
-[store and share](/doc/start/data-and-model-versioning#storing-and-sharing) them
-remotely like any other iteration of the pipeline.
+DVC allows to update the parameters defined in the pipeline without modifying
+the files manually. We use this feature to set the convolutional units in
+`train.py`.
 
 ```dvc
-dvc push
-git push
+$ dvc exp run --set-param model.conv_units=24
+...
+Reproduced experiment(s): exp-7b56f
+Experiment results have been applied to your workspace.
+...
 ```
 
 <details>
 
-### 💡 Important information on storing experiments remotely.
+### Run multiple experiments in parallel
 
-The commands in this section require both a `dvc remote default` and a
-[Git remote](https://git-scm.com/book/en/v2/Git-Basics-Working-with-Remotes). A
-DVC remote stores the experiment data, and a Git remote stores the code,
-parameters, and other metadata associated with the experiment. DVC supports
-various types of remote storage (local file system, SSH, Amazon S3, Google Cloud
-Storage, HTTP, HDFS, etc.). The Git remote is often a central Git server
-(GitHub, GitLab, BitBucket, etc.).
+Instead of running the experiments one-by-one, we can define them to run in a
+batch. This is especially handy when you have long running experiments.
 
-</details>
-
-Experiments that have not been made persistent will not be stored or shared
-remotely through `dvc push` or `git push`.
-
-`dvc exp push` enables storing and sharing any experiment remotely.
+We add experiments to the queue using the `--queue` option of `dvc exp run`. We
+also use `-S` (`--set-param`) to set a value for the parameter.
 
 ```dvc
-$ dvc exp push gitremote exp-bfe64
-Pushed experiment 'exp-bfe64' to Git remote 'gitremote'.
+$ dvc exp run --queue -S model.conv_units=32
+Queued experiment '3cac8c6' for future execution.
+$ dvc exp run --queue -S model.conv_units=64
+Queued experiment '23660b6' for future execution.
+$ dvc exp run --queue -S model.conv_units=128
+Queued experiment '6591a57' for future execution.
+$ dvc exp run --queue -S model.conv_units=256
+Queued experiment '9109ea9' for future execution.
 ```
 
-`dvc exp list` shows all experiments that have been saved.
+Next, run all (`--run-all`) queued experiments in parallel. You can specify the
+number of parallel processes using `--jobs`:
 
 ```dvc
-$ dvc exp list gitremote --all
-72ed9cd:
-        exp-bfe64
+$ dvc exp run --run-all --jobs 2
 ```
 
-`dvc exp pull` retrieves the experiment from a Git remote.
+</details>
 
-```dvc
-$ dvc exp pull gitremote exp-bfe64
-Pulled experiment 'exp-bfe64' from Git remote 'gitremote'.
-```
+## Comparing experiments
```

Comment:
This should go right after the `dvc exp run` ... w/o any subtitle ideally. There is a natural connection and the most important thing:

`dvc exp run` to run ... now to see your staff - check the `dvc exp show` out.

Everything else is secondary in this document and should go in some general sections like:

`Setting params`

If we want to keep that section at all it should be about `--set-param` and about general way - edit file and run.

Reply:
Added a brief example after the first run